cmd/monitor/monitor.go

```diff
@@ -231,21 +236,53 @@ func (m *AppCreateAvailabilityMonitor) createProject() (string, error) {
 			DisplayName: "A temporary test project created by monitor-project-lifecycle app.",
 		},
 	)
-
 	if err != nil {
 		return "", fmt.Errorf("error creating project: %v", err)
 	}
 
+	// Add monitoring annotation and label to the namespace
+	ns, err := m.Client.CoreClient.Namespaces().Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", fmt.Errorf("error getting namespace: %v", err)
+	}
+
+	// TODO: Get decision from deads or jliggit what pattern we should use for the annotation name.
+	//       More info:  https://github.com/openshift/monitor-project-lifecycle/issues/24
+	ns.Annotations[nsAnnotationKey] = nsAnnotationValue
+
+	if ns.Labels == nil {
+		ns.Labels = make(map[string]string)
+	}
+
+	ns.Labels[nsLabelKey] = nsLabelValue
+
+	ns, err = m.Client.CoreClient.Namespaces().Update(ns)
+	if err != nil {
+		return "", fmt.Errorf("error updating namespace: %v", err)
+	}
+
 	return name, nil
 }
 
-// deleteProject deletes the project for a test.
+// deleteMonitorProjects deletes all test apps projects created by this app.
 // TODO: Add prometheus metrics for timing and errors.
-func (m *AppCreateAvailabilityMonitor) deleteProject(namespace string) error {
```

Comment:
This should still return an error to allow the caller to decide how to respond.

Reply:
I believe I've fixed this one. It's not going away because it's on the removal instead of that addition.